api/envoy/config/route/v3/route_components.proto

```diff
@@ -911,7 +912,10 @@ message RouteAction {
   // fires, the stream is terminated with a 408 Request Timeout error code if no
   // upstream response header has been received, otherwise a stream reset
   // occurs.
-  google.protobuf.Duration idle_timeout = 24;
+  google.protobuf.Duration idle_timeout = 24 [
+    (udpa.annotations.security).configure_for_untrusted_downstream = true,
+    (udpa.annotations.security).configure_for_untrusted_upstream = true
```

Comment:
It isn't always necessary to configure this as there is a default idle timeout for all streams. Is this going to push people into a config that they don't need? Thoughts on how to handle this?

Reply:
I think for this and other comments, what we need is the ability to add some additional text to include in the alert box. I'll add a proto format to define what manifests look like and can have protodoc pick this up.

Reply:
For this specific one, on further examination, I think we should eliminate and focus on stream idle defaults.